pkg/container-collection/container-collection.go

```diff
@@ -0,0 +1,213 @@
+// Copyright 2019-2021 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package containercollection provides the ContainerCollection struct to keep
+// track of the set of running containers and primitives to query that set with
+// various criteria.
+//
+// It is used by the Gadget Tracer Manager to keep track of containers part of
+// Kubernetes pods and by Local Gadget Manager to keep track of containers on a
+// Linux system.
+package containercollection
```

Comment:
Nit: missing empty line.

Reply:
godoc requires that the package documentation is not separated by a new line, otherwise it would not pick it up.

It works fine at the moment (running `godoc` and checking on http://localhost:6060/pkg/github.com/kinvolk/inspektor-gadget/pkg/container-collection/):

![Screenshot from 2021-09-22 12-25-00](https://user-images.githubusercontent.com/27575/134327435-b4acb14e-d246-4719-beea-5b08e173e4b3.png)

Reply:
Ah, I thought it was the license text that was before. Sorry for the noise.